src/includes/performance/always-inherit-sampling-decision/native.mdx

```diff
@@ -1,4 +1,4 @@
-<Alert level="warning" title="Sampling Functions">
+<Alert level="warning">
```

Comment:
FYI, the suggestion I'm making about the level of the note being a note and not a warning is based on our style guide, as we try to apply more consistency to the whole of docs:
https://www.notion.so/sentry/Tips-for-Contributors-b1997ea7f5e84eb9a02e0770b6dcc453#9563c2214dfa4a92bca2cc9205129b38

Reply:
Thanks for the link to this, I've updated all of the alerts 👍